exporters/prometheus-dcgm/dcgm-exporter/dcgm-exporter

```diff
@@ -37,8 +37,15 @@ mkdir -p $(dirname ${OUTPUT_FILE})
 trap 'echo "Caught signal, terminating..."' HUP INT QUIT PIPE TERM
 
 if [ "${HOST_ENGINE}" = "yes" ]; then
-    echo "Starting NVIDIA host engine..."
-    nv-hostengine 2> /dev/null
+    nvsdev=`ls -1 /dev/nvidia-nvswitch* 2> /dev/null`
+    if [ ! -z "${nvsdev}" ]; then
+        echo "Warning: Suppressing $0 -e option."
+	echo "  nvidia-fabricmanager will start nv-hostengine on this platform."
+	HOST_ENGINE=no
```

Comment:
You are just resetting the `-e` option which we keep no by default. See this https://github.com/NVIDIA/gpu-monitoring-tools/blob/master/exporters/prometheus-dcgm/dcgm-exporter/dcgm-exporter#L5. If you already have the nv-hostengine running and don't want to start a separate instance, just unset `-e` by overriding the docker entry-point https://github.com/NVIDIA/gpu-monitoring-tools/blob/master/exporters/prometheus-dcgm/dcgm-exporter/Dockerfile#L21

$ docker run -it --entrypoint /usr/local/bin/dcgm-exporter nvidia/dcgm-exporter